esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -2275,6 +2352,56 @@ void CWsDfuEx::doGetFileDetails(IEspContext &context, IUserDescriptor *udesc, co
         }
     }
 
+    if (version >= 1.25)
```

Comment:
Out of curiosity hold old are clients that are < 1.25 ? i.e. what platform version would those client be from?

Reply:
ESP server has to support a client which would like to use any version of ESP.

Reply:
Before HPCC 5.0.0, WsDfu version is < 1.25. If an ESP client would like to use that version of WsDfu, the client may tell ESP that the client version is < 1.25.